packages/grafana-ui/src/components/uPlot/types.ts

```diff
@@ -2,7 +2,11 @@ import React from 'react';
 import uPlot from 'uplot';
 import { DataFrame, FieldColor, TimeRange, TimeZone } from '@grafana/data';
 
-export type NullValuesMode = 'null' | 'connected' | 'asZero';
+export enum NullValuesMode {
+  null = 'null',
```

Comment:
@mckn 

imo, "Gaps" would be more clear here. "display null as null" doesnt really convey what the result would look like.

Reply:
+1  We should change that in the UI.  This property has been around for a long long time, so it is hard to change, but we can certainly change what is displayed in the UI

Reply:
why are you creating a new NullValuesMode object vs the one in https://github.com/grafana/grafana/blob/master/packages/grafana-data/src/types/data.ts#L139

Reply:
Awesome feedback! Will have a look 👍

Reply:
@ryantxu I did not know that we had that one :) But I have replaced it with the one you referenced.

Reply:
+1 for the change in the UI